src/NUnitFramework/framework/Internal/ValueTaskAwaitAdapter.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
```

Comment:
I guess this -ValueTaskAwaitAdapter - should be mentioned in the ticket or PR?

Reply:
It has nothing to do with the ticket, I will extract it into a separate PR.

Reply:
Done. Removed from this PR and Created #4543